fern/pages/sdks/integration/maven-central.mdx

```diff
@@ -7,7 +7,7 @@ To authenticate and integrate Maven Central within your Java SDK, you'll need to
 ## Creating a Maven Central User Token
 <Steps>
 ### Log In
-Log into [Maven Central](https://central.sonatype.com/api/auth/login). 
+Log into the Nexus Repository Manager web interface where you would like to host your project. This is likely [oss.sonatype.org](oss.sonatype.org) or [s01.oss.sonatype.org](s01.oss.sonatype.org). 
```

Comment:
where you host your project*

Reply:
I tried to word it to include both those who have published to maven already, and those who have not